script.js

```diff
@@ -1,4 +1,141 @@
+// BEAT THAT
+// Game mode (mode 1. play dice rolls) (mode 2. decide the dice order) (mode 3. decide the winner)
+// there are two players, two dices
+
+// global variables
+// randomly selected dice for each players
+var player1Dice1 = "";
+var player1Dice2 = "";
+var player2Dice1 = "";
+var player2Dice2 = "";
+
+// the numbers that decided by players
+var player1Num = "";
+var player2Num = "";
+
+//player score
+var player1Score = [];
+var player2Score = [];
+
+// game mode starts with player 1
+var gameMode = "player 1";
```

Comment:
[Comment] Nice job to organise all your global variables together and with comments